client/stores/useGlobalConfiguration.ts

```diff
@@ -0,0 +1,38 @@
+import type { Ref } from 'vue'
+import { defineStore } from 'pinia'
+
+export interface GlobalConfigurationStore {
+  toggleRegistrations: Ref<boolean>
+  defaultDiskQuotaPolicy: Ref<DiskQuotaPolicy>
+  defaultDiskQuota: Ref<number | null>
+  updateConfiguration: (configuration: Partial<GlobalConfiguration>) => void
+  fetchConfiguration: () => void
+}
+
+export const useGlobalConfiguration = defineStore('globalConfiguration', (): GlobalConfigurationStore => {
+  const toggleRegistrations = ref<boolean>(false)
+  const defaultDiskQuotaPolicy = ref<DiskQuotaPolicy>('LIMITED')
+  const defaultDiskQuota = ref<number | null>(524288000)
```

Comment:
```suggestion
  const defaultDiskQuota = ref<number | null>(524288000) // 500.0 MiB
```

Reply:
I dunno, those defaults should be put somewhere.

I also don't like the fact that we have to sync them with the backend, but I don't see how we can avoid that anyway...

Reply:
I agree with you, but I think it is a fairly unique case, so we could leave it like this..